static/_redirects

```diff
@@ -1 +1,4 @@
-/phpoole    /cecil
+/ https://next-design-iteration--jamstack-site.netlify.app/ 301!
+/about https://next-design-iteration--jamstack-site.netlify.app/generators/ 301!
+/contribute https://next-design-iteration--jamstack-site.netlify.app/generators/ 301!
+/* https://next-design-iteration--jamstack-site.netlify.app/generators/:splat 301!
```

Comment:
```suggestion
/* https://next-design-iteration--jamstack-site.netlify.app/generators/:splat 301!/
```
testing something with trailing slash to avoid double 301 redirect chain